ui/src/testing/stubs/leo-apps-api-stub.ts

```diff
@@ -1,12 +1,8 @@
 import { AppsApi } from 'notebooks-generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class LeoAppsApiStub extends AppsApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.